components/MainView.js

```diff
@@ -82,12 +86,19 @@ window.onload = function () {
         }
 
         editList() {
-            const input = prompt("Enter list of values separated by comma (,)", this.props.noCookieMode ? null : localStorage.getItem('itemList'));
+            const restoredItemList = this.props.noCookieMode ?
+                this.state.initialItemList.join(',') : localStorage.getItem('itemList');
```

Comment:
Please format this as 
```js
isX
  ? thenY
  : elseZ
```
So it would be a little bit easier for the eyes to understand this.